polyfills/Number/isNaN/tests.js

```diff
@@ -0,0 +1,27 @@
+it('has correct instance', function () {
+	expect(Number.isNaN).to.be.a(Function);
+});
+
+it('has correct argument length', function () {
+	expect(Number.isNaN.length).to.be(1);
+});
+
+describe('returns true with', function () {
+	it('numbers', function () {
+		expect(Number.isNaN(NaN)).to.be(true);
+		expect(Number.isNaN(Number.NaN)).to.be(true);
+		expect(Number.isNaN(0/0)).to.be(true);
+	});
+});
+
+describe('returns false with', function () {
+	it('numbers', function () {
```

Comment:
incorrectly described tests.  I suggest either drop the `describe`s and call the nested test cases "returns true for NaNs" and "returns false for valid numbers and non-number data types"; or create an `it` for each value, eg returns false with... string, undefined, empty object.